agentdock-core/src/evaluation/runner/index.ts

```
@@ -0,0 +1,464 @@
+        case 'Sentiment': // Added
+          evaluator = new SentimentEvaluator(evalConfig.config);
+          evaluatorType = `${evalConfig.type}:${evalConfig.config.criterionName}`;
+          break;
+        case 'Toxicity': // Added
+          evaluator = new ToxicityEvaluator(evalConfig.config);
+          evaluatorType = `${evalConfig.type}:${evalConfig.config.criterionName}`;
+          break;
+        default:
+          // This will be caught by the exhaustive check in validateEvaluatorConfigs if a new type is added
+          // but not handled here. However, for safety:
+          const _exhaustiveCheck: never = evalConfig;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)